apps/main/src/llamalend/features/borrow/components/BorrowActionInfoAccordion.tsx

```diff
@@ -109,9 +120,13 @@ export const BorrowActionInfoAccordion = <ChainId extends IChainId>({
             loading={ratesLoading || futureRatesLoading}
             testId="borrow-apr"
           />
-          {loanToValue != null && (
-            <ActionInfo label={t`Loan to value ratio`} value={formatPercent(loanToValue)} testId="borrow-ltv" />
-          )}
+          <ActionInfo
+            label={t`Loan to value ratio`}
+            value={loanToValue ? formatPercent(loanToValue) : '...'}
```

Comment:
Maybe we should standardize the value to be in `...` when `loading` is true internally in the `ActionInfo` component 🤔

Reply:
That would be great, I love consistency and I hate repetition